packages/core/src/shared/clients/s3Client.ts

```diff
@@ -450,7 +451,7 @@ export class DefaultS3Client {
                 Bucket: bucket.name,
                 Delimiter: DEFAULT_DELIMITER,
                 MaxKeys: request.maxResults ?? DEFAULT_MAX_KEYS,
-                Prefix: request.folderPath,
+                Prefix: request.folderPath ?? DEFAULT_PREFIX,
```

Comment:
can you add a brief comment that explains what's going on here, and link to https://github.com/aws/aws-toolkit-vscode/issues/4643

Reply:
Addressed on 63b7bed.